docs/deployment.md

```diff
@@ -1,7 +1,152 @@
-# Deployment
+# Deployment examples
 
+[Local GPU deployment](#local-gpu-deployment)
+
+[OpenShift deployment](#openshift)
+
+
+_______
+
+## Local GPU deployment
+
+### Docker compose
+
+Manifest example: [compose-gpu.yaml](./deploy-examples/compose-gpu.yaml)
+
+This deployment has the following features:
+
+  - NVIDIA cuda enabled
+
+Install the app with:
+
+```sh
+docker compose -f docs/deploy-examples/compose-gpu.yaml up -d
+```
+
+For using the API:
+
+```sh
+# Make a test query
+curl -X 'POST' \
+  "localhost:5001/v1alpha/convert/source/async" \
+  -H "accept: application/json" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "http_sources": [{"url": "https://arxiv.org/pdf/2501.17887"}]
+  }'
+```
+
+Requirements:
+
+  - debian/ubuntu/rhel/fedora/opensuse
+  - docker
+  - nivida drivers >=550.54.14
+  - nvidia-container-toolkit 
+
+Docs:
+
+https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/supported-platforms.html
+
+https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#id6
+
+Steps:
+
+1. Check driver version and which GPU you want to use (0/1/2/3.. and update [compose-gpu.yaml](./deploy-examples/compose-gpu.yaml) file or use `count: all`)
+
+```sh
+nvidia-smi
+```
```

Comment:
this block should be indented with 4 spaces. similar the other code blocks in the itemization